src/wiki/development/development-builds.md

```diff
@@ -5,11 +5,12 @@ eleventyNavigation:
   order: 1
 ---
 # Development Builds
+Please understand that **these builds are not intended for most users**. There may bugs, and other instabilities. You have been warned.
 
-There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). These have debug information in the binaries, so their file sizes are relatively larger.
-Portable builds are provided for AppImage on Linux, Windows, and macOS.
+There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). Debug information is present within the binaries, making their file sizes larger than otherwise normal.
+Portable dev-builds are provided for Windows, macOS, and AppImage (Linux).
```

Comment:
Actually, Linux portable builds are back

Reply:
Also MacOS is *not* portable, nor is AppImags

Reply:
I was just going off of what was there before. Are you sure about AppImage though, I thought being portable was the point of them?

Reply:
AppImage is not portable since it saves data in home directories

Reply:
I'll clean that one up then, thank you